RileyLinkKit/Extensions/NSDate.swift

```diff
@@ -8,12 +8,12 @@
 
 import Foundation
 
-internal func ==(lhs: NSDate, rhs: NSDate) -> Bool {
+func ==(lhs: NSDate, rhs: NSDate) -> Bool {
     return lhs === rhs || lhs.compare(rhs) == .OrderedSame
 }
 
-public func <(lhs: NSDate, rhs: NSDate) -> Bool {
+// MARK: - Comparable
+
+func <(lhs: NSDate, rhs: NSDate) -> Bool {
     return lhs.compare(rhs) == .OrderedAscending
 }
-
-extension NSDate: Comparable { }
```

Comment:
Since the only code that leverages this function was comparing `NSDate` explicitly, it didn't need to be declared `Comparable`. I actually ended up removing the only use of this function below, anyway.